content/blog/kubernetes-is-better-with-pulumi/index.md

```diff
@@ -0,0 +1,108 @@
+---
+title: "Kubernetes Is Better with Pulumi"
+date: 2020-06-02
+meta_desc: "Ten reasons why you should deploy modern apps and Kubernetes with Pulumi"
+meta_image: dangerous_pulumi.gif
+authors:
+   - levi-blackstone
+   - sophia-parafina 
+tags:
+    - kubernetes
+---
+
+You’ve containerized your application, and it’s running great on your desktop using Docker Compose or Swarm. But now it’s time to test it locally with minikube and then put it into production with Kubernetes. Manifests are a bit like Compose files - it’s just YAML, right?
+
+<!--more-->
+
+## It’s Dangerous to Go Alone. Take this
+
+Recently, Pulumi engineer [Levi Blackstone](https://twitter.com/levi_blackstone) posted a [Twitter thread](https://twitter.com/levi_blackstone/status/1265798769242550272) on why he thinks Pulumi is the best way to build and deploy modern applications on Kubernetes. Levi might be a bit biased, but he makes solid points for using Pulumi with Kubernetes, whether you’re a developer or operator.
+
+![Legend of Pulumipus](dangerous_pulumi.gif)
+
+## For Devs
+
+First, let’s take a look at Kubernetes from a dev’s point of view.
+
+- **Use the full Kubernetes API surface** – Each of Pulumi’s Kubernetes SDKs (TypeScript, Python, .NET and Go) are generated from the OpenAPI spec. This is critical because it means that resource definitions in any language are a 1:1 match with the upstream API. In the example below, the deployment manifest on the left corresponds to the typescript on the right.
```

Comment:
```suggestion
- **Use the full Kubernetes API surface** – Each of Pulumi’s Kubernetes SDKs (TypeScript, Python, .NET and Go) are generated from the OpenAPI spec. This is critical because it means that resource definitions in any language are a 1:1 match with the upstream API. In the example below, the deployment manifest on the left corresponds to the TypeScript on the right.
```